docs/virtual_machines/creating_vms.md

```diff
@@ -16,6 +16,48 @@ For a full set of flags and their description use the following command:
 virtctl create vm -h
 ```
 
+# Creating Instance Types
+
+The virtctl subcommand `create instancetype` allows easy creation of an Instance type
+manifest from the command line. This manifest can be later used to create your VirtualMachine.
```

Comment:
How can the manifest be used to create a VirtualMachine? What do you mean by that?

Reply:
I will drop this line